dds/DCPS/DataReaderImpl.h

```diff
@@ -529,11 +529,12 @@ class OpenDDS_Dcps_Export DataReaderImpl
     DDS::SampleStateMask sample_states, DDS::ViewStateMask view_states,
     DDS::InstanceStateMask instance_states) = 0;
 
+#endif
+
   virtual void set_instance_state(DDS::InstanceHandle_t instance,
```

Comment:
Changes to this method require changes to `tests/DCPS/unit/MyTypeSupportImpl.cpp`.

Reply:
I'm confused what you mean by changes are needed to the cpp? It looks like that function was stubbed out in the .h. Are you saying you want the function tested in Main.cpp?

Reply:
Yes, I was saying the stub needed to be changed.